pymc/model/core.py

```diff
@@ -2245,3 +2246,12 @@ def normal_logp(value, mu, sigma):
     )
 
     return var
+
+
+def print_value(var, name=None):
```

Comment:
I think this should go in `pymc.pytensorf`

Reply:
I also think so; I'll shift it.
Thanks.